2023-10-16/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+func concatenate(a, b string) string {
+	return a + b
+}
+
+func swap(a, b string) (string, string) {
+	return b, a
+}
+
+func main() {
+	a := "el"
+	b := "Jo"
+
+	a, b = swap(a, b)
+
+	fmt.Println(concatenate(a, b))
```

Comment:
Now when we go back and look for our variables using the concatenate function we will get them in the order of b, a since the swap function are returning them in this order. 

Note: We've not changed the names of the variables, the swap function is just affecting the order of the variables when we ask for them. To prove this i added `fmt.Println(a)` and the second line in the prompt was "Jo"